docs/content/preview/manage/backup-restore/snapshot-ysql.md

```diff
@@ -31,79 +31,102 @@ showAsideToc: true
+When the command is executed, it generates a unique ID for the snapshot, and prints it out:
 
-    ```sh
-    yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> list_snapshots
-    ```
+```output
+Started snapshot creation: 0d4b4935-2c95-4523-95ab-9ead1e95e794
+```
 
-    The output shows the snapshot UUID and the current state.
+You can then use this ID to check the status of the snapshot, [delete it](#deleting-a-snapshot), or use it to [restore the database](#restoring-a-snapshot).
 
-    ```output
-    Snapshot UUID                     State
-    4963ed18fc1e4f1ba38c8fcf4058b295  COMPLETE
-    ```
+The `create_database_snapshot` command exits immidiately, so its completion does not necessarily means that the snapshot is successfully created. Before using the snapshot, you should verify its status by running the [`list_snapshots`](../../../admin/yb-admin/#list-snapshots) command:
 
-1. Create a backup of the YSQL metadata, including the schema, by running the following `ysql_dump --create` command:
+```sh
+yb-admin list_snapshots
+```
 
-    ```sh
-    ysql_dump -h <ip> --masters <ip1:7100,ip2:7100,ip3:7100> --include-yb-metadata --serializable-deferrable --create --schema-only --dbname <database_name> --file ysql.schema.sql
-    ```
+This command will print out all the snapshots that exist in the cluster with their statuses. Locate the ID of the new snapshot and make sure its status is `COMPLETE`:
+
+```output
+Snapshot UUID                         State
+0d4b4935-2c95-4523-95ab-9ead1e95e794  COMPLETE
+```
+
+## Deleting a Snapshot
+
+Snapshots created using the `create_database_snapshot` never expire and are retained as long as the cluster exists. If a snapshot is no longer needed, you can delete it by running the [`delete_snapshot`](../../../admin/yb-admin/#delete-snapshot) command and providing the ID of the snapshot:
+
+```sh
+yb-admin delete_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794
+```
+
+## Restoring a Snapshot
+
+To restore a database to a previously created snapshot, run the [`restore_snapshot`](../../../admin/yb-admin/#restore-snapshot) command and provide the ID of the snapshot:
 
-1. Get the current YSQL schema catalog version by running the [`yb-admin ysql_catalog_version`](../../../admin/yb-admin/#ysql-catalog-version) command:
+```sh
+yb-admin restore_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794
+```
+
+The above command will rollback the database to the state which it had when the snapshot was created. The restore happens in-place, i.e. it changes the state of the existing database within the same cluster.
+
+## Moving a Snapshot to an External Storage
+
+Storing snapshots in-cluster is extermely efficient, but also comes with downsides. First of all, it increases the cost of the cluster - increasing number of snapshot can inflate the space consumption on the storage volumes. Second of all, in-cluster snapshots do not protect you from disaster scenarios like filesystem corruption or hardware failures.
```

Comment:
```suggestion
Storing snapshots in-cluster is extremely efficient, but also comes with downsides. It can increase the cost of the cluster by inflating the space consumption on the storage volumes. Also, in-cluster snapshots don't protect you from a disaster scenario like filesystem corruption or a hardware failure.
```